### ISSUE TITLE
Triggered by increased protein intake

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10552]
**Preferred term label:**

**Synonyms**
Triggered by protein-rich foods

**Definition (free text, please give PubMed ID)**

PMID: 32373372 While the urea cycle disorders (UCDs) classically present in the neonatal stage, they have become increasingly recognized as a rare cause of unexplained hyperammonemic encephalopathy in adults. Many metabolic triggers for late-onset UCDs have been described in the literature including **excessive protein intake**.

PMID: 38484105 Classic IVA is characterized by acute metabolic decompensations (vomiting, poor feeding, lethargy, hypotonia, seizures, and a distinct odor of sweaty feet). Acute metabolic decompensations are typically triggered by fasting, (febrile) illness (especially gastroenteritis), or **increased protein intake**. 

**Parent term (use hpo.jax.org/app)**
Triggered by food ingestion HP:0033793

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:664 Ornithine transcarbamylase deficiency
ORPHA:33 Isovaleric acidemia 

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001067
- label: Triggered by high protein intake
- parents: [Triggered by (HP:0025204)](https://hpo.jax.org/app/browse/term/HP:0025204)
- pmid: PMID:32373372|PMID:38484105
- def: Applies to a sign or symptom that is provoked or brought about ingestion of high quantities of protein.
- comment: Metabolic triggers for late-onset urea cycle disorders include excessive protein intake.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10552
- synonyms: Triggered by protein-rich foods(NONE)
